### PR TITLE
#160577434 setup react router 

### PR DESCRIPTION
**Pull request by ibetween**
#### What does this PR do?
- Setup react router for our client
#### Description of Task to be completed?
- Install and configure the router
-  Create a dummy route for login and the “/” page
- Create a dummy login component
- Create a components directory
#### How should this be manually tested
- Clone the repo: `git clone https://github.com/andela/ah-shakas-frontend`
- Checkout to router branch: `git checkout ch-react-router-160577434`
- Start the app using the following command `yarn start`

#### Any background context you want to provide?
- A router was required for the app

#### What are the relevant pivotal tracker stories?

[#160577434](https://www.pivotaltracker.com/story/show/160577434)

#### Screenshots
- Below is a login route displaying a login page
- When one clicks the login button they will be redirected to the homepage i.e `"/"`
screen shot 2018-10-30 at 14 57 25

### CODE REVIEW

**Comment by Arthur236** [comment]
@james-chege You need to work on the hound errors

**Comment by ibetween** [comment]
> @james-chege You need to work on the hound errors

Thanks for pointing it out @Arthur236. Fixing them ASAP

File: src/components/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {Button, Input, Row} from 'react-materialize';
+
+class LoginPage extends React.Component {
```
[the remark this answers is not in the document]
Comment (ibetween) [reply src/components/LoginPage.jsx]: We can ignore this error because this component will later have state.

**Comment by Arthur236** [comment]
@james-chege I see that some files could not be reviewed due to incorrect configuration in `.eslintrc`. Could you look into this

**Comment by ibetween** [comment]
> @james-chege I see that some files could not be reviewed due to incorrect configuration in `.eslintrc`. Could you look into this

@Arthur236  There was an error in hound configuration. I have communicated with @koechkevin to fix it. It is now fine.

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import HomePage from './components/HomePage.jsx';
```
Comment (Arthur236) [inline src/App.jsx]: The `.jsx` extension is not necessary on the imports.
Kindly remove it for all other imports.

File: src/components/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            />
+          </Row>
+          <Row>
+            <Input
+              type="password"
+              label="password"
+              s={12}
+            />
+          </Row>
+          <Link to="/"><Button waves="light">Login</Button></Link>
```
Comment (Arthur236) [inline src/components/LoginPage.jsx]: The button should not be wrapped in a link

Comment (ibetween) [reply src/components/LoginPage.jsx]: Alright. Will fix this

File: src/App.jsx (hunk 2 above)
Comment (ibetween) [reply src/App.jsx]: Noted 👍

File: src/components/LoginPage.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            />
+          </Row>
+          <Row>
+            <Input
+              type="password"
+              label="password"
+              s={12}
+            />
+          </Row>
+          <Link to="/">Login</Link>
```
Comment (Arthur236) [inline src/components/LoginPage.jsx]: What I meant here was, just have the button alone and not the link

**Comment by lukorito** [comment]
@james-chege how about you remove the package.json file?

File: src/components/LoginPage.jsx (hunk 4 above)
Comment (ibetween) [reply src/components/LoginPage.jsx]: Alright. Let me rectify that.

**Comment by ibetween** [comment]
@lukorito I hope you mean  `package-lock.json`. If it is this file, I have removed it.

**Review by Arthur236 — APPROVED** [review]
Looks good 👍

**Pull request merged**